fe/fe-core/pom.xml

```diff
@@ -172,6 +172,7 @@ under the License.
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```

Comment:
Please define it in `dependency` manage in the parent pom, we need to manage dependencies uniformly.